crypto/ui/ui_lib.c

```diff
@@ -794,6 +807,21 @@ const char *UI_get0_result_string(UI_STRING *uis)
     return NULL;
 }
 
+int UI_get_result_string_length(UI_STRING *uis)
+{
+    switch (uis->type) {
+    case UIT_PROMPT:
+    case UIT_VERIFY:
+        return uis->result_len;
+    case UIT_NONE:
+    case UIT_BOOLEAN:
+    case UIT_INFO:
+    case UIT_ERROR:
+        break;
+    }
+    return -1;
```

Comment:
Will this cause a "not all cases used" warning?  Even if not, other switch statements in this file use a default.